File: programmers/js/07. 수박수박수박수박수박수?.js
function solution(n) {
    let answer = '';

    // 홀수일때 짝수일때
    for(let i=0; i<n; i++){
        answer += i%2 == 0 ? "수" : "박";
    } 

    return answer;
}

console.log(solution(5))
console.log(solution(2))
console.log(solution(3))

File: programmers/java/07. 수박수박수박수박수박수?.java
class Solution {
    public String solution(int n) {
        String answer = "";
        
        for(int i=0; i < n; i++) {
            if(i%2 == 0) {
                answer += "수";
            } else {
                answer += "박";
            }
        }
       return answer;
    }
}

 //public String solution(int n){
 //   String result = "";
 //   for (int i = 0; i < n; i++)
 //     result += i % 2 == 0 ? "수" : "박";
 //       return result;
 //}
